public/video-ui/src/reducers/composerPageReducer.js

```diff
@@ -1,10 +1,10 @@
 export default function usage(state = {}, action) {
   switch (action.type) {
-    case 'VIDEO_PAGE_CREATE_POST_RECEIVE': {
-      const videoId = action.newPage.videoId;
+    case 'usage/addNewlyCreatedVideoUsage': {
+      const videoId = action.payload.videoId;
       state[videoId] = {
-        composerId: action.newPage.composerId,
-        usage: action.newPage.usage
+        composerId: action.payload.composerId,
+        usage: action.payload.usage
```

Comment:
The expected payloads between the nice slice action and this JS file don't seem to match: 

<img width="854" height="371" alt="Image" src="https://github.com/user-attachments/assets/e0a88e9a-74b0-4aef-88a2-b60b116e61b9" />